README.md

```diff
@@ -94,6 +94,10 @@ To add a resource or contribute, please check out our contribution guide [here](
 ## Bots
 
 - [Simple Twitter Bot Tutorial with Node.js](https://blog.shahednasser.com/simple-twitter-bot-tutorial-with-node-js/)
+- [Create a Simple WhatsApp Chatbot without Coding](https://twitter.com/mridulrb/status/1410221692819968006?s=20)
```

Comment:
Instead of including multiple versions please include just one, preferably not the tweet.

Reply:
I have only kept 1 link here.